File: src/tests/stuff/moretest/gcd.go
package main

import "fmt"

func main() {
	fmt.Println("gcd")
	a := 123456789012
	b := 987654321098
	for b != 0 {
		t := b
		b = a % b
		a = t
	}
	fmt.Println(a)
}

File: src/tests/stuff/moretest/gcd.rs
fn main() {
    println!("gcd");
    let mut a: i64 = 123456789012;
    let mut b: i64 = 987654321098;
    while b != 0 { let t = b; b = a % b; a = t; }
    println!("{}", a);
}
